zk_toolbox/crates/zk_inception/src/commands/explorer.rs

```diff
@@ -0,0 +1,265 @@
+    let ecosystem_path = shell.current_dir();
+    //  Keep track of allocated ports (initialized lazily)
+    let mut ecosystem_ports: Option<EcosystemPorts> = None;
+
+    // Get ecosystem level apps.yaml config
+    let apps_config = AppsEcosystemConfig::read_or_create_default(shell)?;
+    // What chains to run the explorer for
+    let chains_enabled = apps_config
+        .explorer
+        .chains_enabled
+        .unwrap_or_else(|| ecosystem_config.list_of_chains());
+
+    // For each chain - initialize if needed or read previously created configs
+    let mut explorer_chain_configs = Vec::new();
+    let mut backend_configs = Vec::new();
+    for chain_name in chains_enabled.iter() {
+        let chain_config = ecosystem_config
+            .load_chain(Some(chain_name.clone()))
+            .ok_or_else(|| anyhow::anyhow!("Failed to load chain config for {}", chain_name))?;
+
+        // Should initialize? Check if apps chain config exists
+        let mut should_initialize = false;
+        let apps_chain_config_path = AppsChainConfig::get_config_path(&ecosystem_path, &chain_name);
+        let apps_chain_config = match AppsChainConfig::read(shell, &apps_chain_config_path) {
+            Ok(config) => config,
+            Err(_) => {
+                should_initialize = true;
+                // Initialize explorer database
+                logger::info(msg_explorer_initializing_database_for(&chain_name));
+                let db_config = fill_database_values_with_prompt(&chain_config);
+                initialize_explorer_database(&db_config).await?;
+
+                // Allocate ports for backend services
+                let services_config =
+                    allocate_explorer_services_ports(&ecosystem_path, &mut ecosystem_ports)
+                        .context(MSG_EXPLORER_FAILED_TO_ALLOCATE_PORTS_ERR)?;
+
+                // Build and save apps chain config
+                let app_chain_config = AppsChainConfig {
+                    explorer: AppsChainExplorerConfig {
+                        database_url: db_config.full_url(),
+                        services: services_config,
+                    },
+                };
+                app_chain_config.save(shell, &apps_chain_config_path)?;
+                app_chain_config
+            }
+        };
+
+        let l2_rpc_url = get_l2_rpc_url(&chain_config)?;
+        let l2_rpc_url = Url::parse(&l2_rpc_url).context("Failed to parse L2 RPC URL")?;
+
+        // Build backend compose config for the explorer chain services
+        let backend_compose_config =
+            ExplorerBackendComposeConfig::new(chain_name, l2_rpc_url, &apps_chain_config.explorer)?;
+        backend_configs.push(backend_compose_config);
+
+        // Read or create explorer chain config (JSON)
+        let explorer_chain_config_path =
+            ExplorerChainConfig::get_config_path(&ecosystem_path, chain_name);
+        let explorer_chain_config = match should_initialize {
```

Comment:
```suggestion
        let explorer_chain_config = if should_initialize {
```

Reply:
looks like I ran zk lint or something else  instead of zk_supervisor